### PR TITLE
Add conditional schedule for TW modules

### PR DESCRIPTION
**Pull request by SergioAtSUSE**
- Verification runs:
  - [Tumbleweed](http://openqa.slindomansilla-vm.qa.suse.de/tests/2485) (`_EXIT_AFTER_SCHEDULE=1`)
  - [SLE15-SP2](http://openqa.slindomansilla-vm.qa.suse.de/tests/2486) (`_EXIT_AFTER_SCHEDULE=1`)
  - [Leap 15.2](http://openqa.slindomansilla-vm.qa.suse.de/tests/2487) (`_EXIT_AFTER_SCHEDULE=1`)
  - [SLE15-SP1](http://openqa.slindomansilla-vm.qa.suse.de/tests/2489) (`_EXIT_AFTER_SCHEDULE=1`)

### CODE REVIEW

**Comment by foursixnine** [comment]
@SergioAtSUSE The verification runs still contain some of the test modules that were supposed to be removed, eg: `test_16_extend_timeout`

**Comment by SergioAtSUSE** [comment]
> @SergioAtSUSE The verification runs still contain some of the test modules that were supposed to be removed, eg: `test_16_extend_timeout`

Yes, 15SP2ORLATER

File: schedule/systemd/suse_patches-systemd_testsuite.yaml [hunk 1]
```diff
@@ -19,12 +46,6 @@ schedule:
...
     - systemd_testsuite/test_15_dropin
-    - systemd_testsuite/test_16_extend_timeout
-    - systemd_testsuite/test_17_udev_wants
-    - systemd_testsuite/test_18_failureaction
-    - systemd_testsuite/test_19_delegate
-    - systemd_testsuite/test_20_mainpidgames
-    - systemd_testsuite/test_21_sysusers
     - systemd_testsuite/test_22_tmpfiles
-    - systemd_testsuite/test_23_type_exec
+    - {{systemd_testsuite_15SP2ORLATER}}
```
Comment (eliroca) [inline schedule/systemd/suse_patches-systemd_testsuite.yaml:50]: Looks like we need some quotation marks here: `- '{{systemd_testsuite_15SP2ORLATER}}'`
Added them in another pull request: https://github.com/os-autoinst/os-autoinst-distri-opensuse/pull/10141
Failing tests:
- [Tumbleweed](https://openqa.opensuse.org/tests/1248884)
- [Leap 15.2](https://openqa.opensuse.org/tests/1248455)

Comment (perlpunk) [reply schedule/systemd/suse_patches-systemd_testsuite.yaml:50]: The quotes were added already in another PR, so rebasing to master should help.

Comment (eliroca) [reply schedule/systemd/suse_patches-systemd_testsuite.yaml:50]: Thanks for the info!

**Pull request merged**